src/material-experimental/mdc-snack-bar/snack-bar-container.ts

```diff
@@ -188,4 +213,37 @@ export class MatSnackBarContainer extends BasePortalOutlet
       throw Error('Attempting to attach snack bar content after content is already attached');
     }
   }
+
+  /**
+   * Starts a timeout to move the snack bar content to the live region so screen readers will
+   * announce it.
+   */
+  private _screenReaderAnnounce() {
+    if (!this._announceTimeoutId) {
+      this._announceTimeoutId = setTimeout(() => {
```

Comment:
This extra timeout will likely break people's tests and might make the PR harder to land. What if this was done at the end of the animation?

Reply:
I had already noticed that is some of the snack bar spec tests. This is very helpful, now I'm subscribing to onEnter to trigger the DOM changes.